token.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
can some of the strings here be moved to the ENV file?

Reply:
No these can't. This is created at run time (though we should be able to move one in). This is where google looks for the token when authenticating